server.py

```diff
@@ -55,14 +57,26 @@ def investigate_issue(request: InvestigateRequest):
         source_instance_id=request.source_instance_id,
         raw=raw_data,
     )
-    return {
-        "analysis": ai.investigate(
-            issue,
-            # TODO prompt should probably be configurable?
-            prompt=load_prompt("builtin://generic_investigation.jinja2"),
-            console=console,
-        ).result
+    investigation = ai.investigate(
+        issue,
+        # TODO prompt should probably be configurable?
```

Comment:
Yes, I think it is worth supporting an override (optional though)
@arikalon1 do you agree?

Reply:
yeah, I agree
We should allow an optional override
@RobertSzefler can you please add it?

Reply:
added